kytos/core/auth.py

```diff
@@ -87,6 +95,76 @@ def _generate_token(cls, username, time_exp):
             algorithm=cls.encode_algorithm,
         )
 
+    def create_user(self):
+        """Create user to database"""
+        req = request.json
```

Comment:
`create_user` is depending on `request.json` that it shouldn't depend on, when you extract this as a method of the new `UserController` let's pass the expected arguments accordingly.